airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -370,19 +374,26 @@ def await_pod_start(
         :param startup_check_interval: Interval (in seconds) between checks
         :return:
         """
-        curr_time = time.time()
-        while True:
+
+        @tenacity.retry(
+            retry=tenacity.retry_if_exception_type(ResourceNotReadyException),
+            wait=tenacity.wait_fixed(startup_check_interval),
```

Comment:
you wrote "exponential" in the title but this is fixed.

personally i don't see any issue with fixed waits for this but it may not be what you intended?

there's also `wait_random` if you are worried about stampeding herd.

Reply:
Ah, yeah, this one specifically took a wait interval so I kept that, the others were hardcoded wait(1) and wait(2)